data/thingengine.builtin.tt

```diff
@@ -29,6 +29,13 @@ class @org.thingpedia.builtin.thingengine.builtin {
   #_[formatted=[{type="text",text="${program}"}]]
   #[doc="retrieve the list of supported commands for the given device"];
 
+  query canned_reply(in req intent: Enum(hello,cool,sorry,thank_you),
+                   out text : String)
+  #_[canonical="get canned reply"]
+  #_[confirmation="a reply to ${intent}"]
```

Comment:
what would be the confirm strategy for easter eggs when they are mixed with thingtalk commands?